<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROIBasedPanel</class>
 <widget class="QWidget" name="ROIBasedPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seed Based Method</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>369</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Templete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sel_templete">
        <item>
         <property name="text">
          <string>AAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Talarich</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Broadman</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="sel_volume_Btn">
        <property name="text">
         <string>Select Volumes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="normalize_Btn">
        <property name="text">
         <string>Normalize</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="en_detrend__Check">
        <property name="text">
         <string>Enable Detrend</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="detrend_Btn">
        <property name="text">
         <string>Detrend</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="en_bandpass_Check">
        <property name="text">
         <string>Band(Hz)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="in_lower_frequency"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>~</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="in_higher_frequency"/>
      </item>
      <item>
       <widget class="QPushButton" name="filter_Btn">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Correlation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sel_correlation">
        <item>
         <property name="text">
          <string>Pearson Correlation</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sel_base_method">
        <item>
         <property name="text">
          <string>Seed Based</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ROI Based</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="run_Btn">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_mask_Btn">
      <property name="text">
       <string>Create Mask</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="load_mask_Btn">
      <property name="text">
       <string>Load Mask</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>461</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
